tests/unit/templates/test_search_index_contract.py

```
@@ -0,0 +1,92 @@
+"""Search-index schema contract consumed by allium-deploy's Pages Function."""
+
+import json
+import re
+
+from allium.lib.aroi_validation import get_v3_search_index_thresholds
+from allium.lib.search_index import PARALLEL_THRESHOLD, generate_search_index
+
+
+def _generate_index(tmp_path, relays_data, filename, use_parallel):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_generate_index`

(ANN202)